services/dynamic-sidecar/Dockerfile

```diff
@@ -196,6 +196,7 @@ COPY --chown=scu:scu --from=prod-only-deps  ${DYNAMIC_SIDECAR_DY_VOLUMES_MOUNT_D
 COPY --chown=scu:scu services/dynamic-sidecar/docker services/dynamic-sidecar/docker
 RUN chmod +x services/dynamic-sidecar/docker/*.sh
 
+# https://docs.docker.com/reference/dockerfile/#healthcheck
```

Comment:
This is correct, the long start period is required, since sometimes it takes a while for the sidecars to start on some autoscaled machines. That's why the start-period is this long.

Reply:
added a note